### PR TITLE
README.md file updated

### PR DESCRIPTION
**Pull request by aparna2071**
I added the command to clone the project at line number 29 and fixed the typo for credentials in line 32 in README.md.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/126"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ Before installation, please make sure you have already installed the following t
 ## 🛠️ Installation Steps
 
 1. Fork the project
-2. Clone the project
+2. Clone the project `git clone https://github.com/{your-username}/good-first-issue-finder.git`
```
Comment (krshkodes) [inline README.md:29]:
```suggestion
2. Clone the project
    ```bash
      git clone https://github.com/YOUR_USERNAME/good-first-issue-finder.git
    ```
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Unreadable, Efficient, Inelegant, Maintainable

**Review by krshkodes — COMMENTED** [review]
Hi 👋 This looks awesome!

But I have this small suggestion:

- We can make the code a block so it is copy-able easily
- And make the {your-username} in CAPS, so it stands out!

Thanks for your awesome contribution!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

File: README.md (hunk 1 above)
Comment (aparna2071) [reply README.md:29]: Thanks @krshkodes for your suggestions.
I will make the necessary changes.

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Comment by aparna2071** [comment]
> ## Looking good to me
> Your suggested changes looking good to me. __ [sema-logo](https://semasoftware.com/gh)  **Summary:** 👌 This code looks good  |  **Tags:** Elegant

Thanks @schmelto for your approval

**Review by emmalearnscode — APPROVED** [review]
Looks great, thanks for your contribution!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Comment by aparna2071** [comment]
Thanks @emmalearnscode for approving and merging

**Pull request merged**